docs/mmteb/points/731.jsonl

```diff
@@ -1,2 +1,2 @@
-{"GitHub": "malteos", "New dataset": 2}
+{"GitHub": "malteos", "New dataset": 6}
```

Comment:
As far as I can tell, the previous PR adds a German retrieval dataset. As German has already been covered in retrieval before, 2 pts is correct, unless I've missed something obvious.
```suggestion
{"GitHub": "malteos", "New dataset": 2}
```